src/themes/classic/components/select.js

```diff
@@ -0,0 +1,43 @@
+import { defaultControlStyles } from '../deprecated/shared'
```

Comment:
If we want to keep `shared` around, feel free to move it under `/components` instead of in `deprecated`!